### PR TITLE
Add RuleSetProvider necessary to get rules executed

### PR DESCRIPTION
**Pull request by travisMiehm**

### CODE REVIEW

File: src/main/kotlin/com/faire/detekt/FaireRulesProvider.kt [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      PreventBannedImports(config),
+      ReturnValueOfLetMustBeUsed(config),
+      UseEntriesInsteadOfValuesOnEnum(config),
+      UseFirstOrNullInsteadOfFind(config),
+      UseMapNotNullInsteadOfFilterNotNull(config),
+      UseOfCollectionInsteadOfEmptyCollection(config),
+      UseSetInsteadOfListToSet(config),
+    )
+  )
+}
```
Comment (Nava2) [inline src/main/kotlin/com/faire/detekt/FaireRulesProvider.kt:59]: 🤣 we should probabyl install detekt to run in this repo

File: src/main/kotlin/com/faire/detekt/FaireRulesProvider.kt [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import io.gitlab.arturbosch.detekt.api.Config
+import io.gitlab.arturbosch.detekt.api.RuleSet
+import io.gitlab.arturbosch.detekt.api.RuleSetProvider
+
+internal class FaireRulesProvider : RuleSetProvider {
+  override val ruleSetId = "FaireRuleSet"
+
+  override fun instance(config: Config): RuleSet = RuleSet(
+    ruleSetId,
+    listOf(
```
Comment (Nava2) [inline src/main/kotlin/com/faire/detekt/FaireRulesProvider.kt:34]: arg names

**Review by Nava2 — APPROVED** [review]

File: src/main/kotlin/com/faire/detekt/FaireRulesProvider.kt (hunk 1 above)
Comment (travisMiehm) [reply src/main/kotlin/com/faire/detekt/FaireRulesProvider.kt:59]: TODO() :(

File: src/main/kotlin/com/faire/detekt/FaireRulesProvider.kt (hunk 2 above)
Comment (travisMiehm) [reply src/main/kotlin/com/faire/detekt/FaireRulesProvider.kt:34]: I don't think they're necessary here. The two arguments are pretty clear.

**Pull request merged**